man/systemd.network.xml

```diff
@@ -3768,6 +3768,18 @@ ServerAddress=192.168.0.1/24</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RapidCommit=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When true, the server supports
+          <ulink url="https://datatracker.ietf.org/doc/html/rfc4039">RFC 4039</ulink>. When a client send
```

Comment:
"client sends a DHCPDISCOVER message"
"then the server will reply with a DHCPACK message"